core/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/Collector.scala

```diff
@@ -131,6 +131,20 @@ trait Collector {
         }
       }
 
+    def xForwardedProto(routes: Route): Route =
+      if (!collectorConf.ssl.redirect) routes
```

Comment:
The collectorConf.ssl also has a `enable` parameter.  I dislike this `enable` parameter immensely but for legacy reasons I wonder if we should respect it.

```
if (collectorConf.ssl.redirect.enable && collectorConf.ssl.redirect)
  optionalHeaderValueByName("X-Forwarded-Proto") { .... }
else routes
```

Reply:
good point, I'll respect that for now and we can remove that later